plugins/memcached.yaml

```diff
@@ -18,17 +17,32 @@ parameters:
     default: end
 
 # Set Defaults
-# {{$file_path := default "/var/log/memcached.log*" .file_path}}
 # {{$start_at := default "end" .start_at}}
 
 # Pipeline Template
 pipeline:
-  - id: memcached_input
-    type: file_input
+  - id: memcached_journald_input
+    type: journald_input
     include:
```

Comment:
`include` is not an accepted parameter for `journald_input`. I'm kinda surprised this isn't failing

Reply:
Yeah I didn't remove it from file_input good catch. I will update it to use directory.